src/modules/dashboard/components/Dialogs/GnosisControlSafeDialog/GnosisControlSafeForm.tsx

```diff
@@ -0,0 +1,158 @@
+    defaultMessage: 'Select transaction',
+  },
+  buttonTransaction: {
+    id: `dashboard.GnosisControlSafeDialog.GnosisControlSafeForm.buttonTransaction`,
+    defaultMessage: 'Add another transaction',
+  },
+  buttonInteract: {
+    id:
+      'dashboard.GnosisControlSafeDialog.GnosisControlSafeForm.buttonInteract',
+    defaultMessage: 'Interact',
+  },
+});
+
+const displayName = 'dashboard.GnosisControlSafeDialog.GnosisControlSafeForm';
+
+export interface GnosisSafe {
+  name: string;
+  address: Address;
+  chain: string;
+}
+
+interface Props {
+  colony: Colony;
+  safes: GnosisSafe[];
+  back?: () => void;
+}
+
+const renderAvatar = (address: string, item) => (
+  <Avatar
+    seed={address.toLocaleLowerCase()}
+    size="xs"
+    notSet={false}
+    title={item.name}
+    placeholderIcon="at-sign-circle"
+  />
+);
+
+const GnosisControlSafeForm = ({
+  back,
+  handleSubmit,
+  safes,
+  isSubmitting,
+  isValid,
+}: Props & FormikProps<FormValues>) => {
+  return (
+    <>
+      <DialogSection>
+        <Heading
+          appearance={{ size: 'medium', margin: 'none', theme: 'dark' }}
+          text={MSG.title}
+        />
+      </DialogSection>
+      <DialogSection appearance={{ theme: 'sidePadding' }}>
+        <FormattedMessage
+          {...MSG.description}
+          values={{
+            a: (chunks) => (
+              <ExternalLink href={GNOSIS_SAFE_INTEGRATION_LEARN_MORE}>
+                {chunks}
+              </ExternalLink>
+            ),
+          }}
+        />
+      </DialogSection>
+      <DialogSection>
+        <div className={styles.safePicker}>
+          <SingleSafePicker
+            appearance={{ width: 'wide' }}
+            label={MSG.selectSafe}
+            name="safe"
+            filter={filterUserSelection}
+            renderAvatar={renderAvatar}
+            data={safes}
+            showMaskedAddress
+            disabled={isSubmitting}
+            placeholder={MSG.safePickerPlaceholder}
+          />
+        </div>
+      </DialogSection>
+      <DialogSection appearance={{ theme: 'sidePadding' }}>
+        <Select
+          options={transactionOptions}
+          label={MSG.transactionLabel}
+          name="transactionType"
+          appearance={{ theme: 'grey', width: 'fluid' }}
+          placeholder={MSG.transactionPlaceholder}
+          disabled={isSubmitting}
+        />
+      </DialogSection>
+      <DialogSection>
+        <div className={styles.addTransaction}>
+          <AddItemButton text={MSG.buttonTransaction} disabled />
+        </div>
+      </DialogSection>
+      <DialogSection appearance={{ align: 'right', theme: 'footer' }}>
+        <Button
+          appearance={{ theme: 'secondary', size: 'large' }}
+          onClick={back}
+          text={{ id: 'button.back' }}
+        />
+        <Button
+          appearance={{ theme: 'primary', size: 'large' }}
+          onClick={() => handleSubmit()}
+          text={MSG.buttonInteract}
```

Comment:
![Screenshot from 2022-06-23 22-13-05](https://user-images.githubusercontent.com/1193222/175379260-6d8a135e-9b62-4f2a-8a06-e031b46835e2.png)

This is one for product again, I think this should say either "Create Transaction" or "Initiate Transaction" since that's what you're actually doing here.

"Interact" seems very vague to me